src/server/routes/routes.ts

```diff
@@ -0,0 +1,189 @@
+import fs from 'fs';
+import { Router, Request } from 'express';
+import type { IncomingHttpHeaders } from 'http';
+
+import getTemplateData from '../data-access';
+import ServiceNames from '../../common/service-names';
+import renderTemplate from '../render-template';
+import { processOrientationOption } from '../../browser/helpers';
+import generatePdf, { previewPdf } from '../../browser';
+import { SendingFailedError, PDFNotFoundError } from '../errors';
+import config from '../../common/config';
+import { OPTIONS_HEADER_NAME } from '../../common/consts';
+import { PreviewReqQuery } from '../../common/types';
+
+const PORT = config.webPort;
+
+type PreviewOptions = unknown;
+type PreviewHandlerRequest = Request<
+  PreviewOptions,
+  any,
+  unknown,
+  PreviewReqQuery
+>;
+
+export type GenerateHandlerReqyest = Request<
+  unknown,
+  unknown,
+  Record<string, any>,
+  { service: ServiceNames; template: string }
+>;
+
+export type HelloHandlerRequest = Request<
+  unknown,
+  unknown,
+  unknown,
+  { policyId: string; totalHostCount: number }
+>;
+
+export interface PupetterBrowserRequest
+  extends Request<
+    unknown,
+    unknown,
+    unknown,
+    { service: ServiceNames; template: string }
+  > {
+  headers: IncomingHttpHeaders & {
+    [OPTIONS_HEADER_NAME]?: string;
+  };
+}
+
+const APIPrefix = '/api/crc-pdf-generator/v1';
```

Comment:
How about populating this from env variable?

Reply:
Again I would probably move this to the config variable as ENV variables are not really a thing. Maybe I should do the same for the prod/dev flags.